### DISCUSSION TITLE
Marginal Spatial Std. Dev. Going to 0

### DISCUSSION THREAD

**Discussion by jmhatch** [sdmTMB/sdmTMB#369] [General]
Hi,

I'm fitting Beta regressions to data that represent fractions derived from continuous time, so $y \in (0,1)$. For now, I'm splitting the data into day versus night and fitting two separate regressions because we're interested in diel differences. To keep things simple, I'm just fitting an intercept model with a random effect for individual and spatial + spatiotemporal random fields (not sure that is really simple, but I just mean no covariates 😄).

$$
\mu_{i,s,t} = \text{logit}^{-1} \left(\beta_0 + b_i +  \mathbf{\omega_{s}} + \mathbf{\epsilon_{s,t}} \right)
$$

Everything looks good when fitting the regression to the day data, but something interesting happens when fitting the regression to the night data. The marginal spatial standard deviation goes to 0, but there is still a decent amount of variability across individuals and spacetime. Does this mean that, on average, there isn't much spatial variation in the $y\text{'s}$ at night? Is this something I should be concerned with, like maybe it's a modeling artifact of some kind (like decisions made around mesh construction)?

**Comment by ericward-noaa** [comment]
Good question. Yes, when any of the marginal SDs are going to 0, it generally implies that the model would be better off with this component removed. I think this can be a good thing, if it's leading you toward a simpler model. I don't think it'd necessarily mean that there isn't spatial variation in the ys at night -- because you still have the spatiotemporal effects in there. What it does mean I think is that there doesn't seem to be much support for a common pattern shared across time slices (e.g. years) that you use with spatiotemporal effects.

**Reply by jmhatch** [reply]
Thanks, @ericward-noaa! That makes sense. If I'm understanding right, basically the data aren't supporting an overall spatial pattern outside of what is already being captured in the spatiotemporal dynamics. Appreciate the insight.

**Comment by ericward-noaa** [comment]
Yep exactly!
